core/runtime.rs

```diff
@@ -577,6 +600,9 @@ impl JsRuntime {
     &mut self,
     cx: &mut Context,
   ) -> Poll<Result<(), AnyError>> {
+    // We always poll the inspector if it exists.
+    let _ = self.inspector().map(|i| i.poll_unpin(cx));
```

Comment:
This result should not be ignored, instead it should alter result of `poll_event_loop`